tests/test_filterset.py

```diff
@@ -687,33 +689,50 @@ def setUp(self):
 
     def test_fields_set(self):
         class F(FilterSet):
+            o = OrderingFilter(fields=('username', 'status'))
```

Comment:
I would create separate test methods for ordering - too much is under test here.